index.html

```diff
@@ -2,6 +2,8 @@
 ---
 {% include header.html %}
 
+<a href="javascript:void(0);" id="rocketmeluncur" class="showrocket" ><i></i></a>
```

Comment:
why dummy i tag  `<i></i>` is present ?

Reply:
Can't you see that <i> tag has a command in css file. If it's not there some parts of animation are missing!

Reply:
bette use `id` istead of dummy tags